proposals/lip-0048.md

```diff
@@ -275,7 +286,7 @@ def beforeCrossChainCommandExecute(trs: Transaction, ccm: CCM) -> None:
 #### After Cross-chain Command Execution
 
 ```python
-def afterCrossChainCommandExecute(trs: Transaction, ccm: CCM) -> None:
+def afterCrossChainCommandExecution(trs: Transaction, ccm: CCM) -> None:
```

Comment:
In this function the `Token.burn` should also be replaced with more complicated logic.

Reply:
Thanks for this comment, I was not sure whether this should also be changes. Could it be that `messageFeeTokenID` is different from `TOKEN_ID_FEE` (I suppose yes)? If yes, then the remaining CCM fee should be transferred to a different account address, not the `ADDRESS_FEE_POOL`, right?
If answers to the above questions are yes, then my suggestion for the account address for the remaining CCM fee is `ADDRESS_CCM_FEE_POOL`. Is this OK with you?

Reply:
I think, it is possible to have different token IDs for message fee and for fee. However I think it would make more sense to have a single fee pool for command fees and ccm fees, there is no problem with having different types of tokens in one account.

Reply:
I wasn't sure about this too... but now that you mention, of course it is so. OK, later during the day I will correct the code.

Reply:
Indeed there is no problem having different tokens in the same account, it's same as user Alice having in her account LSK tokens and sidechain tokens.

Corner case, that came to my mind, do we have also to initialize user account for this address and specified token? This is related to the topic of initializing accounts, but holds also for the account of `ADDRESS_FEE_POOL`. (perhaps @sergeyshemyakov knows better)

Reply:
Thanks again for the clarification, Greg!

Reply:
I think the accounts should be automatically initialized in the [Token.transfer](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0051.md#transfer-2) function.
I guess it could be annoying from the user perspective if they provide not enough fee, but this is our protocol.

Reply:
@sergeyshemyakov that means that in the very first transaction, the user should know about the extra fee needed for initializing the account for this address? And same for the first ccm?